website/src/css/custom.css

```diff
@@ -59,13 +66,17 @@
 .button {
```

Comment:
Remove these `.button` styles. Use the default by Docusaurus CSS. Refer to https://github.com/facebook/create-react-app/blob/master/docusaurus/website/src/pages/index.js as an example of the CSS classes available.